content/api/commands/intercept.md

```diff
@@ -265,7 +249,7 @@ cy.wait('@someRoute').then((interception) => {
 })
 ```
 
-You can chain [`.its()`](/api/commands/its) and [`.should()`](/api/commands/should) to assert against request/response cycles:
+##### You can chain [`.its()`](/api/commands/its) and [`.should()`](/api/commands/should) to assert against request/response cycles:
```

Comment:
I was given direction to make examples like this directly linkable. This might help:

```suggestion
##### Assert against request/response cycles:
```

Reply:
I'll create an issue for level-5 header style (used for linking examples).

Reply:
yeah fair, linking is good. but the style is wack. new issue sounds good... it should probably block this. just a minor font tweak though

Reply:
@amirrustam any objection to taking these L5 headers out until the styling is fixed?

Reply:
reverted all changes where L5 header was added since they look like crap and they're not linkable anyway (see DX-86)